### PR TITLE
Make www/utils.py pylint-compatible

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: scripts/ci/pylint_todo.txt [hunk 1]
```diff
@@ -5,5 +5,4 @@
 ./airflow/models/dagrun.py
 ./airflow/models/taskinstance.py
 ./airflow/stats.py
-./airflow/www/utils.py
-./airflow/www/views.py
+=./airflow/www/views.py
```
Comment (mik-laj) [inline scripts/ci/pylint_todo.txt]:
```suggestion
./airflow/www/views.py
```

File: airflow/www/utils.py [hunk 2]
```diff
@@ -181,7 +181,7 @@ def is_current(current, page):
 
 def epoch(dttm):
     """Returns an epoch-type date"""
-    return int(time.mktime(dttm.timetuple())) * 1000,
+    return (int(time.mktime(dttm.timetuple())) * 1000,)
```
Comment (turbaszek) [inline airflow/www/utils.py:184]: I think it's quite strange that this function return single item tuple. Can't we return just the int value?

File: airflow/www/utils.py [hunk 3]
```diff
@@ -253,6 +253,7 @@ def nobr(attr):
     return nobr
 
 
+# pylint: disable=invalid-name
 def datetime_f(attr_name):
     """Returns a formatted string with HTML for given DataTime"""
     def dt(attr):
```
Comment (turbaszek) [inline airflow/www/utils.py]:
```suggestion
    def dt(attr):  # pylint: disable=invalid-name
```
Should this do? I prefer inline comment as it's easier to understand where exactly is the problem

File: airflow/www/utils.py (hunk 2 above)
Comment (potiuk) [reply airflow/www/utils.py:184]: I just tried to minimise the actual changes - but I can take a look at this as well.

Comment (potiuk) [reply airflow/www/utils.py:184]: the method was unused. Deleted it.

File: airflow/www/utils.py (hunk 3 above)
Comment (potiuk) [reply airflow/www/utils.py]: Agree.

File: airflow/www/utils.py (hunk 2 above)
Comment (turbaszek) [reply airflow/www/utils.py:184]: I'm not sure... https://github.com/apache/airflow/blob/f2da6b419fde71d335d133031f2c0fac28b87289/airflow/www/views.py#L1757

Should we be able to use `datetime.timestamp()`?

Comment (potiuk) [reply airflow/www/utils.py:184]: I jsut added comment :)

**Review by turbaszek — APPROVED** [review]

**Pull request merged**